bitnami/postgresql/README.md

```diff
@@ -276,7 +276,7 @@ $ kubectl delete pvc -l release=my-release
 | `metrics.service.annotations`                 | Additional annotations for metrics exporter pod                                                                                                           | `{}`                        |
 | `metrics.service.loadBalancerIP`              | loadBalancerIP if redis metrics service type is `LoadBalancer`                                                                                            | `""`                        |
 | `metrics.serviceMonitor.enabled`              | Set this to `true` to create ServiceMonitor for Prometheus operator                                                                                       | `false`                     |
-| `metrics.serviceMonitor.additionalLabels`     | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus                                                                     | `{}`                        |
+| `metrics.serviceMonitor.additionalLabels`     | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus  , the value is evaluated as a template.                                                                   | `{}`                        |
```

Comment:
I think that you don't need to include this in the Readme. I took a look another charts like Redis and we are not including this information. So, please remove it. Thanks.